pages/index.js

```diff
@@ -1,64 +1,23 @@
 import styled from "styled-components";
 import { useState } from "react";
-import Link from "next/link";
 
-const Header = styled.header`
-  font-family: monda;
-  font-size: 2em;
-  color: #e7e4d8;
-  font-weight: bold;
-  text-align: center;
-  background-color: #78c9cc;
-`;
-
-const Form = styled.form`
-  color: black;
-  display: grid;
-  width: 70%;
-  margin: 5px auto;
-  padding: 10px;
-  border-radius: 5px;
-  text-align: center;
+const Fieldset = styled.fieldset`
+  height: 70px;
+  margin: 30px;
+  padding: 60px;
 `;
-
 const Label = styled.label`
-  display: center;
-  padding: 20px;
-  font-size: 16pt;
-  text-align: center;
+  font-size: 10pt;
```

Comment:
i searched for it, i was not shure because you are using it often but the consens is "Not rec­om­mended" https://www.w3.org/Style/Examples/007/units.en.html

Reply:
Hey Sascha!i would of course use the recommended version :D thank you!